### DISCUSSION TITLE
NodeResizer, how to prevent resize height?

### DISCUSSION THREAD

**Discussion by Cashmeiser** [xyflow/xyflow#2868] [Q&A]
I've tried out the new NodeResizer in v11 on my CustomNode.
My node have a fixed height which shouldn't change, only the Node width.

Is it possible to prevent/reset the size in one direction?

I can store the initial size with `useStore` which I've tried to enforce to the params in `OnResize` but that doesn't changes anything.

BRs Simon

**Comment by ysds** [comment]
I have the same problem. It is possible to limit resizing by line, but not by handle.
https://codesandbox.io/s/zen-wind-w9znpf?file=/ResizableNode.js

**Reply by chrtze** [reply]
Hey, thanks for the hint. This seems to be a bug. I have created an issue to make this possible: https://github.com/wbkd/react-flow/issues/2986 

In the meantime, you could use some workaround that styles the line control so that it looks like a handle:
https://codesandbox.io/s/gifted-maria-dyjprj?file=/ResizableNode.js
